app/models/user.rb

```diff
@@ -628,6 +635,7 @@ def tag_keywords_for_search
     "#{employer_name}#{mostly_work_with}#{available_for}"
   end
 
+  # TODO: this can be removed once we migrate away from ES
   def search_score
     counts_score = (articles_count + comments_count + reactions_count + badge_achievements_count) * 10
     score = (counts_score + tag_keywords_for_search.size) * reputation_modifier
```

Comment:
`tag_keywords_for_search` is a string made of the content of the job related fields. Not sure why it was in the algorithm, but since we're moving those fields away, this logic can't work now.

https://github.com/forem/forem/blob/ba54a8cf08868568d8f74fd799e50385e8ab5b24/app/models/user.rb#L627-L629